modules/ex/src/clj/exoscale/ex/test.clj

```diff
@@ -17,12 +17,66 @@
                     :actual nil})
       (catch ~k d#
         (t/do-report {:type :pass
-                      :message ~msg,
+                      :message ~msg
                       :expected '~form
-                      :actual d#})
-        (::ex/exception d#))
```

Comment:
first of all, I don't think it works. I don't see this field in the data...do I understand correctly, that this data will be within the meta?
https://github.com/exoscale/ex/blob/master/modules/ex/src/clj/exoscale/ex.cljc#L133

also do we really need to send it at the end of the form?..don't we care only about `do-report`?